src/unix/os390-syscalls.h

```diff
@@ -49,7 +49,7 @@ struct epoll_event {
 typedef struct {
   QUEUE member;
   struct pollfd* items;
-  unsigned long size;
+  int size;
```

Comment:
Just curious, why int?

Reply:
ahh, my bad, `long` seems to be a more appropriate choice.

Reply:
Actually, the maximum number of file descriptors supported by poll on z/OS is 65535.
So int should suffice.